build.sbt

```diff
@@ -1,5 +1,7 @@
 import akka.projections.Dependencies
 
+sonatypeProfileName in ThisBuild := "com.lightbend"
```

Comment:
not sure if it would be better to define it in project/Common.scala? in globalSettings or projectSettings? and if so how to import sonatypeProfileName there?

Reply:
we have
```
organization := "com.lightbend.akka"
```

and the error message was "Profile com.lightbend.akka is not found", but Alpakka Kafka build is using "com.typesafe" with org "com.typesafe.akka"

Reply:
It could live in a common settings for projects that get published, though I guess it wouldn't hurt to define it for all projects as part of global settings.